src/LineageTree/lineageTree.py

```diff
@@ -2115,8 +2101,12 @@ def unordered_tree_edit_distance(
         if delta is None or not callable(delta):
 
             def delta(x, y, corres1, corres2, times):
-                if x is None or y is None:
-                    return 1
+                if x is None:
+                    return times[corres2[y]]
+                if y is None:
+                    return times[corres1[x]]
+                if x is None and y is None:
+                    return 0
```

Comment:
I don't think you version is correct to be honest.
Not saying mine was but for yours, you are doing something like that:
if $(x = \text{None} \land y \neq \text{None}) \lor (x \neq \text{None} \land y = \text{None})$ -> you get some tree length $L_{x | y}>1$, either the length of $y$ ($L_y$) or $x$ ($L_x$)
if $x = \text{None} \land y = \text{None}$ -> you get 0 (no cost alignment)
but if you get
$x \neq \text{None} \land y \neq \text{None}$ then you get
$\frac{|L_x - L_y|}{L_x + L_y}<1$, a cost not comparable of the one when one of the tree is None which does not make sense.

You can see it as $L_{x|y}$ has a time dimension and $\frac{|L_x - L_y|}{L_x + L_y}<1$ is dimensionless